api/HTMLSelectElement.json

```diff
@@ -1069,7 +1069,7 @@
               "version_added": true
             },
             "chrome_android": {
-              "version_added": null
+              "version_added": false
```

Comment:
https://github.com/mdn/browser-compat-data/pull/3732 sets `setCustomValidity` support to true, can be confirmed by `HTMLSelectElement.prototype.setCustomValidity` or `document.createElement('select').setCustomValidity` manually.

Don't know what's gone wrong here, it's in the Chromium's IDL:
https://cs.chromium.org/chromium/src/third_party/blink/renderer/core/html/forms/html_select_element.idl?l=56&rcl=aae10192b9f96a766448a4e48570d67c775b9d63

Reply:
This looks like it may have possibly been a bad merge, since Chrome is set to true, and I would not set the other browsers independently unless stated as not exposed in Webview, which isn’t the case for this.  Fixing this now!